app/assets/javascripts/qpixel_api.js

```diff
@@ -393,17 +381,20 @@ window.QPixel = {
   },
 
   getJSON: async (uri, options = {}) => {
+    const { headers = {} } = options ?? {};
+
     return QPixel.fetchJSON(uri, {}, {
       ...options,
+      headers: {
+        'Accept': 'application/json',
+        ...headers,
+      },
```

Comment:
Should we include the `X-Requested-With` header in here too?

Reply:
Yeah, we should, missed it actually